atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -155,6 +163,12 @@ private void cas(Client client, Long oldVal, long newVal) {
         }
     }
 
+    private String getCurrentValues(CASResult result) {
+        List<Long> timestamps = result.current_values.stream().map(col -> PtBytes.toLong(col.getValue())).collect(
+                Collectors.toList());
```

Comment:
Yep, I noticed your comment a bit further down - for some reason, they didn't resolve. Didn't notice originally (I was going through on the train.